content/projects/katas/level-2/_index.md

```diff
@@ -2,29 +2,39 @@
 _db_id: 637
 content_type: project
 flavours:
-- any_language
+  - any_language
 from_repo: projects/katas/level-1
 learning_outcomes:
-- code_algorithmic_thinking
-- code_pseudo_code
-- code_problem_decomposition
-- code_control_flow
-- code_naming_convention
-- code_code_documentation
-- code_existing_code_update
-- code_basic_calculation
+  - code_algorithmic_thinking
+  - code_pseudo_code
+  - code_problem_decomposition
+  - code_control_flow
+  - code_naming_convention
+  - code_code_documentation
+  - code_existing_code_update
+  - code_basic_calculation
 prerequisites:
   hard:
-  - projects/katas/level-1
+    - projects/katas/level-1
 ready: true
 story_points: 3
 submission_type: continue_repo
 tags:
-- algorithms
-- clean-code
+  - algorithms
+  - clean-code
 title: Level 2 coding challenges
 ---
 
+## Project directory structure
+
+Task 2.1 and Task 2.2 are a updates of Task 4 and 5 from {{% contentlink path="projects/katas/level-1"%}}. You should not create new files for them.
+
+For Task 2.3, the method/function name should be `task2.3`. You are expected to create a new files with the following names:
+
+- `task2.3.js` for javascript
```

Comment:
rather use underscores. The dot specifies an extension, this doesn't cause bugs but it's best to use things for their intended purpose.  The extension `3.js` is not really a thing. 

It's a bit clunky but I think task2_3.js is better because it is clear what part of the file name is the name and which part is the extension.